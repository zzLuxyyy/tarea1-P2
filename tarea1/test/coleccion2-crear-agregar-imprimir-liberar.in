crearColeccion

# creo cuatro libros para ingresarlos a la Coleccion

crearFecha 01/01/2002
crearLibro 849550144 "Juventud" "J.M." "Coetzee" "Un joven sudafricano busca sentido en su vida y vocación en Londres" 4
agregarEnColeccion

crearFecha 01/01/1971
crearLibro 204578912 "Las venas abiertas de América Latina" "Eduardo" "Galeano" "Un ensayo crítico sobre la historia de la explotación en América Latina" 7
agregarEnColeccion

crearFecha 28/01/1813
crearLibro 111213141 "Orgullo y prejuicio" "Jane" "Austen" "Elizabeth Bennet enfrenta el amor y los prejuicios de su época" 1
agregarEnColeccion

crearFecha 01/01/1980
crearLibro 222324252 "El nombre de la rosa" "Umberto" "Eco" "Un fraile investiga misteriosos asesinatos en una abadía medieval" 3
agregarEnColeccion

imprimirColeccion
liberarColeccion
Fin
