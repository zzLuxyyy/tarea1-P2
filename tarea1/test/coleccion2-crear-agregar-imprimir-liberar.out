1>2>#  creo cuatro libros para ingresarlos a la Coleccion.
3>4>5>6>7>8>9>10>11>12>13>14>15>Libro 849550144
Título: Juventud
Autor: J.M. Coetzee
Descripción: Un joven sudafricano busca sentido en su vida y vocación en Londres
Género: 4
Fecha de edición: 1/1/2002
Libro 222324252
Título: El nombre de la rosa
Autor: Umberto Eco
Descripción: Un fraile investiga misteriosos asesinatos en una abadía medieval
Género: 3
Fecha de edición: 1/1/1980
Libro 204578912
Título: Las venas abiertas de América Latina
Autor: Eduardo Galeano
Descripción: Un ensayo crítico sobre la historia de la explotación en América Latina
Género: 7
Fecha de edición: 1/1/1971
Libro 111213141
Título: Orgullo y prejuicio
Autor: Jane Austen
Descripción: Elizabeth Bennet enfrenta el amor y los prejuicios de su época
Género: 1
Fecha de edición: 28/1/1813
16>17>Fin.
